test/basic-querying.test.js

```diff
@@ -1067,6 +1067,19 @@ describe('basic-querying', function() {
         });
       });
     });
+
+    it('should work for or operator', function() {
+      const query = {
+        where: {
+          or: [{name: 'Paul McCartney'}, {email: 'abc@xyz.com'}],
+        },
+      };
+      return User.findOne(query, function(err, user) {
+        should.not.exist(err);
```

Comment:
```suggestion
        if (err) return done(err);
```

Even better, write the test as an `async` function and use `await User.findOne(query)` instead.